tests/test_typings.py

```diff
@@ -44,7 +44,7 @@ def test_run_mypy_tests(self) -> None:
               f'{test_failures}  * * * *****\n'
               f'***** * * *   See {test_out_file_name} for error log  * * * *****'
               )
-        self.assertLessEqual(test_failures, 150, f'mypy on test {test_file}')
+        self.assertLessEqual(test_failures, 150, f'mypy run on tests')
```

Comment:
If we don't want to make this a hard test, I think we can just remove this line.